connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -461,12 +461,24 @@ export async function confluenceCheckAndUpsertPageActivity({
     return false;
   }
 
+  let parents: [string, ...string[], string];
```

Comment:
If you want a tuple types that require at least 3 strings, this is not correct, you should do this:

```suggestion
  let parents: [string, string, ...string[]];
```

TypeScript cannot guarantee the last element would be a string when using a rest element in the middle.